src/rvsdg/egglog_optimizer/mod.rs

```diff
@@ -37,11 +38,16 @@ pub fn rvsdg_egglog_schedule() -> String {
     // but it helps substitutions go through since
     // they take many iterations.
 
-    "(run-schedule
+    "
+    (run-schedule
         (repeat 5 (saturate fast-analyses)
                   (run)
-                  (saturate subst)))"
-        .to_string()
+                  (saturate subst))
+        (repeat 4 subst-beneath)
```

Comment:
Ooh can't say I love this schedule- does it have to be this complex, with constants like 4 and 2?
Is this because we are not allowed to saturate subst-beneath because it might not terminate?

Reply:
Also is this indentation weird?

Reply:
Changed it to be less complex:
```
        (repeat 5 (saturate fast-analyses)
                  (run)
                  (saturate subst))
        (repeat 6 subst-beneath (saturate fast-analyses))
```

The indentation is meant to have future changes produce less diff. Just made it a bit less weird.